server_api.py

```diff
@@ -609,12 +612,12 @@ def store_thumbnail_image(file_path,thumbnail_path):
 
 def get_img_meta(img_id):
     try:
-        db = mysql()
-        db.connect()
-        sql = 'select * from image_data where img_is_delete = 0 and img_id = "%s"' % img_id
-        return db.query(sql)[0]
+        with ImageDao() as imageDao:
+            img_data = ImageDao.getImgData(imgId=img_id)
+        if img_data != None:
```

Comment:
There are 2 places in the server.py that call get_img_meta()(line 155, 176)
I think it would cause problems when get_img_meta() return None.